views/js/checkout-component-renderer.js

```diff
@@ -634,4 +636,12 @@ jQuery(document).ready(function() {
     function isPlaceOrderInProgress() {
         return placeOrderInProgress;
     }
+
+    // When these buttons are clicked, do not close the payment method div
```

Comment:
what do you think instead of listing the fields that won't trigger the collapse method if the collapse would only work for the payment name and the arrow like: 
![image](https://user-images.githubusercontent.com/42297201/110931465-949b4180-832a-11eb-9ba4-6ce1294e61c9.png)

And not the rest below. (Probably introducing some extra margin between the payment method title and the content would make it a bit more visible

Reply:
Html structure updated